pom.xml

```diff
@@ -239,7 +239,7 @@
         <json-smart.version>2.4.8</json-smart.version>
         <junit.version>4.13.2</junit.version>
         <lang-tag.version>1.5</lang-tag.version>
-        <libpam4j.version>1.11</libpam4j.version>
+        <libpam4j.version>5.1.0</libpam4j.version>
```

Comment:
Looks like 8.0.0-M2 is the latest

https://mvnrepository.com/artifact/org.glassfish.main.libpam4j/libpam4j/8.0.0-M2

Reply:
Yes but unfortunately 8.0.0-M2 is incompatible on the binary level as it's compiled with java11 language level.